### DISCUSSION TITLE
O que está faltando ou o que não precisava ter?

### DISCUSSION THREAD

**Discussion by huogerac** [evolutio/djavue3#38] [Q&A]
## 👋 Bem-vindo ao Djàvue, nos ajude a deixá-lo melhor!

Estamos utilizando o 'Discussions' deste repo aqui para entender melhor as dificuldades de quem começou ou tentou utilizá-lo
  * O que faltou ou o que você teve que implementar que poderia ter no template?
  * O que tem no template e você preciso remover?
  * Qualquer informação sobre sua experiência será importante para a evolução deste template 💪

Para iniciar, basta deixar um comentário aqui em abaixo ⬇️.

**Comment by Gabriels999** [comment]
Ter a opção de criar um projeto que usa django templates como front (atualmente não temos escolha)

**Comment by Alfareiza** [comment]
Deploy para o Heroku sería legal

**Reply by huogerac** [reply]
Hey @Alfareiza 
Tinha isto funcionando, mas depois que o heroku passou a cobrar o postgres, focamos mais no Fly.io
Mas obrigado pela sugestão! acho q vale sim ter algo neste sentido!
